section2/exercises/comparisons.js

```diff
@@ -110,22 +121,26 @@ var age = 1;
 
 // YOU DO:
 // Determine if the dog loves to play and loves treats
+console.log(lovesToPlay && lovesTreats);
 
 
 // Determine if the dog loves to play and loves the dog park
+console.log(lovesToPlay && lovesDogPark);
 
 
 // Determine if the dog loves to play or loves the dog park
+console.log(lovesToPlay || lovesDogPark);
 
 
 // Determine if the dog loves to play and is a puppy
+console.log(lovesToPlay && isAPuppy);
 
 // What did your final line of code evaluate to? Why do you think that is? Explain.
-// ANSWER:
+// ANSWER: It came us a a refernece error because isAPuppy is not defined anywhere so it cant determine if this condition is true of false.
```

Comment:
Good reason. I would've liked to see you use a conditional operator like `age > 1` instead of trying to reference a variable that wasn't defined in this exercise. Play around with this a bit more on your own.